config/jetscape_master.xml

```diff
@@ -261,11 +272,11 @@
    <JetHadronization>
        <name>colored/colorless/hybrid</name>
        <take_recoil>1</take_recoil>
-       <eCMforHadronization>2510</eCMforHadronization>
-       <reco_Elevelmax>3</reco_Elevelmax>
+       <eCMforHadronization>460</eCMforHadronization>
```

Comment:
Do these need to be updated? If there is a good reason to do it we can, however if not, we should keep them fixed (and it will break the automated tests)